src/fluree/db/conn/memory.cljc

```diff
@@ -35,63 +36,60 @@
                     {:status 500 :error :db/invalid-db}))))
 
 (defn- write-data!
-  [data-atom data]
+  [store data]
   (let [json (json-ld/normalize-data data)
-        hash (crypto/sha2-256-normalize json)
-        path hash]
-    #?(:cljs (when platform/BROWSER
-               (.setItem js/localStorage hash json)))
-    (swap! data-atom assoc hash data)
+        hash (crypto/sha2-256 json)
+        {path :k
+         size :size}
+        (store/write store hash data)]
     {:name    hash
      :hash    hash
      :json    json
      :size    (count json)
      :address (memory-address path)}))
 
 (defn write-commit!
-  [data-atom commit-data]
-  (write-data! data-atom commit-data))
+  [store commit-data]
+  (write-data! store commit-data))
 
 (defn- read-address
-  [data-atom address]
+  [store address]
   (let [addr-path (address-path address)]
-    #?(:clj  (get @data-atom addr-path)
-       :cljs (or (get @data-atom addr-path)
-                 (and platform/BROWSER (.getItem js/localStorage addr-path))))))
+    (store/read store addr-path)))
 
 (defn- read-data
-  [data-atom address]
-  (let [data (read-address data-atom address)]
+  [store address]
+  (let [data (read-address store address)]
     #?(:cljs (if (and platform/BROWSER (string? data))
                (js->clj (.parse js/JSON data))
                data)
        :clj  data)))
 
 (defn read-commit
-  [data-atom address]
-  (read-data data-atom address))
+  [store address]
+  (read-data store address))
 
 (defn write-context!
-  [data-atom context-data]
-  (write-data! data-atom context-data))
+  [store context-data]
+  (write-data! store context-data))
 
 (defn read-context
```

Comment:
I didn't realize I left these functions in here. Not for this pr, but we should probably clean these up at some point.